### DISCUSSION TITLE
Add End of Round Functionality to Wintertodt Plugin

### DISCUSSION THREAD

**Discussion by liosrakia** [runelite/runelite#16522] [Ideas]
Hi there!
This might be a bit of a niche request, but if there is a  way to have an idle notification on the Wintertodt plugin for when the round ends that would be super useful.

 I'm not sure if the use-case matters, but for me, after I do two full inventories of logs I tend to go afk until the next round starts -- this sometimes leaves me in a situation where I accidentally stay in place, starting the next round low HP and having to leave anyway instead of resetting for the next round starts! 
 
Thanks for reading. :)

**Comment by geheur** [comment]
If there is a chatbox message when the game ends you can put that test into chat notifications plugin to notify you.

**Reply by liosrakia** [reply]
Thank you Sir, I'll give it a shot 🙂
